skema/skema-rs/skema/src/services/mathml.rs

```diff
@@ -60,6 +63,25 @@ pub async fn get_math_exp_graph(payload: String) -> String {
     dot_representation.to_string()
 }
 
+/// Parse a presentation MathML representation of an equation and
+/// return the corresponding LaTeX representation
+#[utoipa::path(
+request_body = String,
+responses(
```

Comment:
Can you fix the indentation of the parentheses over here? It looks a bit strange?